evaluator/src/main/kotlin/Rule.kt

```diff
@@ -113,7 +113,7 @@ abstract class Rule(
         object : RuleMatcher {
             override val description = "hasLabel(${listOfNotNull(label, value).joinToString()})"
 
-            override fun matches() = ruleSet.ortResult.hasLabel(label, value)
+            override fun matches() = ruleSet.ortResult.hasLabel(label, value, false)
```

Comment:
Looking at this commit only, it's unclear what the benefit is to use `hasLabel` instead of `getLabelValues` in `labelContains`. And it's probably hard to find an argument for the commit message, as the next commit even removes `labelContains`. So why bother changing it here in the first place?

Reply:
Boolean arguments should be named for clarity.